package.json

```diff
@@ -266,6 +266,7 @@
     "@typescript-eslint/eslint-plugin": "^3.4.0",
     "@typescript-eslint/eslint-plugin-tslint": "^3.4.0",
     "@typescript-eslint/parser": "^3.4.0",
+    "address": "^1.2.2",
```

Comment:
for some weird reason, this core library is randomly missing if you install Webpack on an M1 mac. Oh well, threw it into the `devDependencies` and called it a day.